contracts/Bridge.sol

```diff
@@ -379,12 +381,13 @@ contract Bridge is Pausable, AccessControl, SafeMath {
         uint72 nonceAndID = (uint72(depositNonce) << 8) | uint72(chainID);
         Proposal storage proposal = _proposals[nonceAndID][dataHash];
 
+        require(proposal._status != ProposalStatus.Inactive, "Proposal is not active");
```

Comment:
Do you think it would make sense to combine these three requires into one statement?
 `require(proposal._status == ProposalStatus.Active || proposal._status == ProposalStatus.Passed , "...");`

Reply:
Agreed. I'll update accordingly.